cmd/image-builder/main.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/labstack/gommon/log"
```

Comment:
external imports tend to go in the 2nd paragraph, it's a bit of a mess in this file though.

Reply:
I noticed, we use an automated tool to fix them all `goimports` it does this kind of sorting for us. I guess I will propose a new PR soon.

Reply:
right, can you move the gommon/log down though?

Reply:
Done.